elementary/monitor/alerts/source_freshness_alert.py

```diff
@@ -153,8 +153,8 @@ def error_message(self) -> str:
     @property
     def summary(self) -> str:
         if self.original_status == "runtime error":
-            return f"Failed to calculate the source freshness of `{self.source_name}`"
-        return f"Freshness exceeded the acceptable times on source `{self.source_name}`"
+            return f'Failed to calculate the source freshness of "{self.source_name}"'
```

Comment:
Do we want to change it from `this format` to "just quotes"?

Reply:
Since the only place it's currently used is the title, and the title is plain test, this had to be done